tests/test_naver_map_crawler.py

```
@@ -0,0 +1,91 @@
+    mock_ua_class.return_value = mock_ua_instance
+    mock_ua_module.UserAgent = mock_ua_class
+    
+    mock_stealth_module = MagicMock()
+    
+    with patch.dict(sys.modules, {
+        'fake_useragent': mock_ua_module, 
+        'playwright_stealth': mock_stealth_module
+    }):
+        yield mock_ua_instance, mock_stealth_module
+
+def test_get_random_ua_installed(mock_installed_packages):
```

Ruff (0.15.0)

[warning] Unused function argument: `mock_installed_packages`

(ARG001)